sklearn/utils/metaestimators.py

```diff
@@ -10,11 +10,47 @@
 __all__ = ['if_delegate_has_method']
 
 
+def hasattr_nested(obj, attr):
+    """Check if obj has a (possibly nested) attribute
```

Comment:
Sorry I fail to understand. When does this happen and how is it related to this PR?